csrc/cpu/attention.cpp

```diff
@@ -428,33 +469,48 @@ void paged_attention_v1(
     const int64_t blocksparse_vert_stride, const int64_t blocksparse_block_size,
     const int64_t blocksparse_head_sliding_step) {
   TORCH_CHECK(kv_scale == 1.0f);
-  TORCH_CHECK(blocksparse_vert_stride <= 1,
-              "CPU backend does not support blocksparse attention yet.");
-  VLLM_DISPATCH_FLOATING_TYPES(query.scalar_type(), "paged_attention_v1_impl",
-                               [&] {
-                                 CPU_KERNEL_GUARD_IN(paged_attention_v1_impl)
-                                 CALL_V1_KERNEL_LAUNCHER_BLOCK_SIZE(scalar_t);
-                                 CPU_KERNEL_GUARD_OUT(paged_attention_v1_impl)
-                               });
+  if (kv_cache_dtype == "auto") {
+    TORCH_CHECK(blocksparse_vert_stride <= 1,
+                "CPU backend does not support blocksparse attention yet.");
+    VLLM_DISPATCH_FLOATING_TYPES(
+        query.scalar_type(), "paged_attention_v1_impl", [&] {
+          CPU_KERNEL_GUARD_IN(paged_attention_v1_impl)
+          CALL_V1_KERNEL_LAUNCHER_BLOCK_SIZE(scalar_t, scalar_t, false);
+          CPU_KERNEL_GUARD_OUT(paged_attention_v1_impl)
+        });
+  } else if (kv_cache_dtype == "fp8") {
```

Comment:
It'd be better to align with the general interface. We now support `fp8`, `fp8_e4m3` and `fp8_e5m2`. Note that `fp8` defaults to `fp8_e4m3`. Please cover all cases and throw an error for unsupported cases.